examples/create-react-app/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "react-scripts build",
```

Comment:
removed since we never need to build the examples and don't want to wait for this when running `yarn build`